src/components/web-form-renderer.vue

```diff
@@ -359,8 +359,9 @@ html, body {
         "body": "The data you entered is valid, but it was not submitted because this is a Form preview."
       },
       "submissionModal": {
-        "title": "Submission successful",
-        "body": "Your data was submitted."
+        "title": "Form successful sent!",
+        "body": "You can fill this Form out again or close if you’re done.",
+        "fillOutAgain": "Fill out again"
```

Comment:
I usually put button text inside an `"action": {` block in the JSON so that it automatically gets a comment about how it's used.